Composer/packages/server/src/services/telemetry.ts

```diff
@@ -69,7 +69,7 @@ if (instrumentationKey) {
 
       // remove PII
       for (const property of piiProperties) {
-        if (data.baseData.properties[property]) {
+        if (typeof data.baseData.properties[property] !== 'undefined') {
```

Comment:
This could be if (data.baseData.properties[property] !== undefined)

The only typeof that will equal 'undefined' is an object set to undefined.